packages/evm-test-data/.gitignore

```
@@ -0,0 +1,14 @@
+# Compiler files
+cache/
+out/
+
+# Ignores development broadcast logs
+/broadcast/
+/broadcast/*/31337/
```

```suggestion
/broadcast/*/912559/
```
This number is the chain id. Using these patterns will ensure we keep the `broadcast` directory, but not commit the contents.

---

fixed this